src/internal/modules/charge-information/forms/use-abstraction-data.js

```diff
@@ -2,16 +2,37 @@
 
 const { get } = require('lodash');
 const Joi = require('@hapi/joi');
+const moment = require('moment');
 
 const { formFactory, fields } = require('shared/lib/forms/');
 const routing = require('../lib/routing');
 const { getActionUrl } = require('../lib/form-helpers');
 
+const getAddionalChoices = (chargeVersions) => {
+  const choices = [{ divider: 'or' }];
+  chargeVersions.map(cv => {
+    if (cv.status === 'current' || cv.status === 'superseded') {
+      choices.push(
+        { value: cv.id,
+          label: `Use charge information valid from ${moment(cv.dateRange.startDate).format('D MMMM YYYY')}`
```

Comment:
Possibly a UCD question, but supposing there are multiple charge versions starting on the same date, how can the user distinguish them?

Possibly we should consider filtering to only show `current` charge versions.  This would mean there is only ever one per start date and the radio options would then be unique.  This might make sense because otherwise we are allowing the user to copy charge versions that somebody has already determined had errors.

Note after `2874` is done, there will be lots of `current` charge versions and not many `superseded`